### PR TITLE
Fixed bug when deleting letters on outcome

### PR DESCRIPTION
**Pull request by giacomolicari**

### CODE REVIEW

File: src/components/SectionOutcomeCategorical/index.js [hunk 1]
```diff
@@ -11,9 +11,6 @@ const OutcomeCategorical = () => (
           name="outcomes"
           component={OutcomeList}
           label="Outcomes"
-          validate={(val) => {
-            return val && val.filter(v => v.length > 0).length > 0 ? undefined : 'Field is required'
-          }}
```
Comment (andre-meyer) [inline src/components/SectionOutcomeCategorical/index.js]: How did you test this? I think the `required` validator will allow empty entries like this: `['Outcome A', 'Outcome B', '']`

Comment (andre-meyer) [reply src/components/SectionOutcomeCategorical/index.js]: It should also have atleast 2 entries, right? Maybe we should add that validation as well.d

If the required validator does allow empty entries, we could also just normalize it before we save, by filtering out all empty entries: https://redux-form.com/7.0.4/docs/api/field.md/#-code-normalize-value-previousvalue-allvalues-previousallvalues-gt-nextvalue-code-optional-

Comment (giacomolicari) [reply src/components/SectionOutcomeCategorical/index.js]: Yes, I re-check this.

Comment (giacomolicari) [reply src/components/SectionOutcomeCategorical/index.js]: Required validator requires the input string to have more than 0 chars, so we would add min-length validator as well.

**Pull request merged**